client/src/style/_config.scss

```diff
@@ -111,5 +111,8 @@ $button-disabled-background: #b2b2b2;
     }
 }
 
+/* Search Bar Config */
+$searchbar-border: #dcdcdc;
```

Comment:
Unsure why this border needs to be common styling if we have only a single search bar component. Maybe remove for now and if we implement a differently styled search bar then we can add it back?

Reply:
seemed like @meganniu preferred all colour hex codes to be defined in _config.scss and used as global variables everywhere else since i got that as a comment in several of my PRs, but i may have misunderstood. could you confirm, megan?

Reply:
Yeah, i'd rather put colors in a common file. If the designers want to generalize the search bar border to input field borders, then we can just rename the variable.